src/components/SubsystemHealth/SubsystemHealth.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
SubsystemHealth component should go under Dashboard folder

Reply:
:+1: